### PR TITLE
Increase padding in buttons on language selection page

### PR DESCRIPTION
**Pull request by SouvikDuttaChowdhury**
merged after #56

### CODE REVIEW

**Comment by SouvikDuttaChowdhury** [comment]
Hey @MichaelKMalak review this branch. This should fix issue #24.

File: lib/features/survey/lauguage_selection_page.dart [hunk 1]
```diff
@@ -22,37 +24,44 @@ class _LanguageSelectionPageState extends State<LanguageSelectionPage> {
...
-                RaisedButton(
-                  child: Padding(
-                    padding: const EdgeInsets.all(8.0),
-                    child: Text(S.of(context).back),
+                Expanded(
+                  child: ButtonBorderedWithBackArrow(
+                    onPressed: () {
+                      Navigator.pop(context);
+                    },
+                    previousButtonLabel: "Back",
```
Comment (MichaelKMalak) [inline lib/features/survey/lauguage_selection_page.dart:47]: try to use `S.of(context).back` instead

File: lib/features/survey/lauguage_selection_page.dart [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/material.dart';
```
Comment (MichaelKMalak) [inline lib/features/survey/lauguage_selection_page.dart:1]: for launcher icon errors, try to run `flutter pub run flutter_launcher_icons:main` on the terminal.

**Review by MichaelKMalak — APPROVED** [review]

**Comment by MichaelKMalak** [comment]
@odhlen ready to merge that..

**Pull request merged**